srcpkgs/ngx_headers_more/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'ngx_headers_more'
+pkgname=ngx_headers_more
+version=0.33
+revision=1
+_nginx_version=1.18.0
+wrksrc=nginx-${_nginx_version}
+build_style=gnu-makefile
+make_build_args="modules"
+hostmakedepends="pcre-devel"
+makedepends="${hostmakedepends}"
+depends="nginx"
+short_desc="Nginx module to set and clear input and output headers"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="BSD-2-Clause"
+homepage="https://github.com/openresty/headers-more-nginx-module"
+distfiles="https://nginx.org/download/nginx-${_nginx_version}.tar.gz
+ https://github.com/openresty/headers-more-nginx-module/archive/v${version}.tar.gz"
+checksum="4c373e7ab5bf91d34a4f11a0c9496561061ba5eee6020db272a17a7228d35f99
+ a3dcbab117a9c103bc1ea5200fc00a7b7d2af97ff7fd525f16f8ac2632e30fbf"
+
+do_configure() {
+	local cfgdir=/etc/nginx
+	local tmpdir=/var/tmp/nginx
```

Comment:
What are these variables for?

Reply:
I feel like something is pretty wrong here regarding cross builds, this just copied the nginx template with removing all the actual cross platform setup required.
Wouldn't be surprised if the cross compiled binaries use wrong sizes etc.